dm_control/autowrap/autowrap.py

```diff
@@ -15,7 +15,7 @@
 
 r"""Automatically generates ctypes Python bindings for MuJoCo.
 
-Parses mjdata.h, mjmodel.h, mjrender.h, mjvisualize.h, mjxmacro.h and mujoco.h;
+Parses mjdata.h, mjmodel.h, mjrender.h, mjvisualize.h, mjxmacro.h, mjui.h and mujoco.h;
```

Comment:
Let's split these onto multiple lines in order to make them a bit easier to parse, e.g.:

```
Parses the following MuJoCo header files:

  mjdata.h
  mjmodel.h
  mjrender.h
  mjui.h
  mjvisualize.h
  mjxmacro.h
  mujoco.h

and generates the following Python source files:

  constants.py:  constants
  ...
```

Reply:
No problem, fixed in the next commit.